deploy/charts/openebs-monitoring/dashboards/cStor/cStor-overview.json

```diff
@@ -833,19 +810,18 @@
       {
         "allValue": null,
         "current": {},
-        "datasource": null,
-        "definition": "query_result(sum(container_memory_working_set_bytes{namespace!=\"\"}) by (namespace))",
+        "datasource": "$datasource",
+        "definition": "label_values(openebs_pool_status, namespace)",
         "description": null,
         "error": null,
         "hide": 0,
-        "includeAll": true,
-        "label": "Namespace",
+        "includeAll": false,
```

Comment:
why this change?

Reply:
Earlier All option was included just so that namespace variable doesn't randomly choose any ns and show in the dashboard. That would have given the user the feeling that OpenEBS is installed in that ns. That's why all option was added. But here we get only the OpenEBS ns therefore this can be omitted.